### ISSUE TITLE
Concerns

### ISSUE THREAD

**Issue by ezehlivinus** [Adaobi-Chuks/UML-Diagrams-for-an-E-Learning-App#3]
On your UseCase Diagram, you made the following comment/note: 
_The student has all the functionality of the visit or while the tutor has the functionality of both the student and the visitor._

You did not show this using [UML generalisation](https://stackoverflow.com/questions/57068530/is-my-use-case-diagram-correct-about-use-case-generalization). With this, you do not need to make the comment.

It is well :)

**Comment by Adaobi-Chuks** [comment]
Good day and compliments of the season. 
Sorry for not replying earlier, I just saw your comment yesterday.

Regarding the issue with my use-case diagram that you noted, the reason I wasn’t able to do that was because I exceeded the 60 maximum  features I was allowed to use on lucid chart. 
So the best alternative was to use a note to indicate that.

I would also appreciate it if you note any other mistakes I made. Thanks.

**Comment by Adaobi-Chuks** [comment]
Good day and compliments of the season. 
Sorry for not replying earlier, I just saw your comment yesterday.

Regarding the issue with my use-case diagram that you noted, the reason I wasn’t able to do that was because I exceeded the 60 maximum  features I was allowed to use on lucid chart. 
So the best alternative was to use a note to indicate that.

I would also appreciate it if you noted other issues I made cause you called this one issue 3 meaning there are other issues.
